### PR TITLE
init project

### PR DESCRIPTION
**Pull request by Ori-Amzallag-Moveo**
I've added photos for the assets dic and typography files.

### CODE REVIEW

File: src/assets/yosi-shitrit.jpg
Comment (Itay6) [inline src/assets/yosi-shitrit.jpg:1]: Change it to a component which get the src and name of chef and can render it automatically.

File: src/styles/desktop-typography.scss [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+/* ---------------- fonts mixins ---------------- */
+
+
+@mixin h1FontDesktop {
+    font-family: primaryFont;
+    font-weight: 200;
+    font-size: 32px;
+    // line-height: 40px;
```
Comment (Itay6) [inline src/styles/desktop-typography.scss]: Remove Comment

File: src/styles/desktop-typography.scss [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    letter-spacing: 1.97px;
+}
+
+@mixin h2FontDesktop {
+    font-family: primaryFont;
+    font-weight: 200;
+    font-size: 30px;
+    // line-height: 35px;
```
Comment (Itay6) [inline src/styles/desktop-typography.scss]: Same, Remove comment

File: src/styles/mobile-typography.scss [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    // line-height: 35px;
+    letter-spacing: 2px;
+}
+
+@mixin iconLabelMobile {
+    font-family: primaryFont;
+    font-weight: 200;
+    font-size: 18px;
+    // line-height: 30px;
+    letter-spacing: 1.97px;
```
Comment (Itay6) [inline src/styles/mobile-typography.scss]: Work with 5's

**Comment by Itay6** [comment]
According our talk, change folder structure and scss vars.

**Comment by Ori-Amzallag-Moveo** [comment]
In my last update i've deleted yosi's photo and rearranged the scss files.
I've used helper function for better readability and modularity.
There is 2 attached photos with the main changes.
Screenshot 2024-07-08 at 14 23 42
Screenshot 2024-07-08 at 14 25 29

File: src/scss/_helpers.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    letter-spacing: $spacing;
+}
+
+@mixin fontProperties($weight, $size, $letterSpacing) {
+    @include fontStyle($primaryFont, $weight, $size, $letterSpacing);
+}
+
+@mixin fontWithSize($weight, $mobileSize, $desktopSize, $letterSpacing) {
+    @include fontProperties($weight, $mobileSize, $letterSpacing);
+    @include small-screen  {
```
Comment (Itay6) [inline src/scss/_helpers.scss]: A more appropriate name would reflect that it is setting styles for larger screens (large-screen, desktop-screen)

**Review by Itay6 — APPROVED** [review]

**Pull request merged**